src/components/no-bank/fakeData.js

```diff
@@ -0,0 +1,12 @@
+export const fakeExpenses = [
```

Comment:
Can we organize this test stuff into a separate folder or clarify that it's for test purposes?